lib/cluster-providers/bottlerocket-cluster-provider.ts

```diff
@@ -16,11 +38,16 @@ export class BottlerocketClusterProvider implements ClusterProvider {
             version: version,
         })
 
-        // TODO add support for customizing.
+        // Allow Context To have instances sizing
+        const instanceTypes = valueFromContext(scope, INSTANCE_TYPE_KEY, DEFAULT_INSTANCE_TYPES);
```

Comment:
Let's make it similar to EC2 Cluster provider and leverage EC2ProviderClusterProps here as well using the same pattern. I assume the code could be centralized and we could derive BRClusterProvider from EC2 cluster provide and just override the node group provisioning.

Reply:
So should we open an issue on this and fix it in the future.